php/libraries/NDB_Caller.class.inc

```diff
@@ -361,8 +400,12 @@ class NDB_Caller
                 $_REQUEST['sessionID']
             );
         }
-        if (method_exists($menu, 'save')) {
+        if (method_exists($menu, 'save') && self::shouldCallSave()) {
```

Comment:
Can't do it that way. Also, `shouldCallSave` doesn't use any instance variables. Hence, static. `shouldCallSave()` isn't used with just menus.